.drone.yml

```diff
@@ -47,10 +46,9 @@ steps:
     - apt-get -y update && apt-get -y install make curl tar
     - make ARCH=arm64 k8s-binaries
     when:
-      instance:
-        - drone-publish.rancher.io
       event:
         - tag
```

Comment:
Should pull request run on drone-pr.rancher.io while tag should run on drone-publish?

Reply:
I thought this was already done automatically and we don't even need this? This (https://github.com/rancher/cri-dockerd/blob/master/.drone.yml) doesn't have `instance:` and works fine for PR and tag.

Reply:
Insteresting, good to know this